src/Purger/MongoDBPurger.php

```diff
@@ -5,12 +5,26 @@
 namespace Doctrine\Common\DataFixtures\Purger;
 
 use Doctrine\ODM\MongoDB\DocumentManager;
+use LogicException;
 
 /**
  * Class responsible for purging databases of data before reloading data fixtures.
  */
 final class MongoDBPurger implements MongoDBPurgerInterface
 {
+    /**
+     * Purge the collections using deleteMany(). Don't create them.
+     */
+    public const PURGE_MODE_DELETE = 1;
+
+    /**
+     * Drop the collections when purging, then recreate them.
+     */
+    public const PURGE_MODE_DROP = 2;
```

Comment:
should this be an enum instead of integer constants in this class ? This is exactly the use case for enums. (ORMPurger uses class constants because enums were not available yet)

Reply:
Thanks for the suggestion. The package requires PHP 8.1 so we can use enums. The type is improved that way.